src/daemon/gateway.go

```diff
@@ -538,30 +538,15 @@ func (gw *Gateway) Spend(wltID string, password []byte, coins uint64, dest ciphe
 }
 
 // CreateWallet creates wallet
-func (gw *Gateway) CreateWallet(wltName string, options wallet.Options) (*wallet.Wallet, error) {
+func (gw *Gateway) CreateWallet(wltName string, options wallet.Options, scanN uint64) (*wallet.Wallet, error) {
```

Comment:
put scanN in options?

Reply:
or separate CreateWalletScanAhead method?

Reply:
The CreateWallet in gateway always scan ahead addresses, so no need to have a separate method.